packages/spark-core/package.json

```diff
@@ -26,6 +26,7 @@
     "chai": "^4.1.2",
     "jsdom": "^11.12.0",
     "nyc": "^11.9.0",
+    "proxyquire": "^2.1.0",
```

Comment:
Why the new need for another dependency? Should this been in devDependencies?

Reply:
proxyquire lets unit tests attach stubs to internal dependancies. I'm using it to have more isolated unit tests. I originally had it in devDependencies, but the linter likes it better in dependencies, not sure why.